### PR TITLE
Add Wolf.bet

### PR DESCRIPTION
**Pull request by SUCHiDEV**

### CODE REVIEW

**Review by Carlgo11 — DISMISSED** [review]
Please move the site to the betting category.

**Comment by SUCHiDEV** [comment]
> Please move the site to the betting category.

Sorry, done

File: _data/betting.yml [hunk 1]
```diff
@@ -19,3 +19,11 @@ websites:
...
     facebook: win2dayat
     lang: de
+    
+  - name: Wolf.bet
+    url: https://wolf.bet/
+    img: wolfbet.png
+    tfa:
+      - totp
+      - email
+    doc: https://wolf.bet/user/settings?tab=2fa
```
Comment (phallobst) [inline _data/betting.yml]: The doc link requires the user to be logged in, so it's not publicly available. They do however have a YouTube video regarding 2FA:
```suggestion
    doc: https://www.youtube.com/watch?v=CgAWzFIwNOk
```

There is no mention of email 2FA. Do you have any more information about that? Can you provide screenshots of the setup process? (With all personal information redacted, of course.)

Also, the icon file size is larger than necessary. Please replace it with this one:
wolfbet

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/betting.yml (hunk 1 above)
Comment (SUCHiDEV) [reply _data/betting.yml]: fixed

**Review by phallobst — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**